src/components/votes/votesTab.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { translate } from 'react-i18next';
+import BoxV2 from '../boxV2';
+import AccountVisual from '../accountVisual';
+import VotesTableHeader from './votesTableHeader';
+import TableRow from '../toolbox/table/tableRow';
+import SpinnerV2 from '../spinnerV2/spinnerV2';
+import { InputV2 } from '../toolbox/inputsV2';
+import styles from './votesTab.css';
+
+class VotesTab extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      showing: 30,
+      filterValue: '',
+    };
+
+    this.onShowMore = this.onShowMore.bind(this);
+    this.handleFilter = this.handleFilter.bind(this);
+  }
+
+  onShowMore() {
+    const max = this.props.votes.length;
+    const showing = (this.state.showing + 30 > max) ? max : this.state.showing + 30;
+    this.setState({ showing });
+  }
+
+  handleFilter({ target }) {
+    this.setState({
+      filterValue: target.value,
+    });
+  }
+
+  render() {
+    const { t, votes, loading } = this.props;
+    const { filterValue } = this.state;
```

Comment:
I think we need to check this with @slaweet the next Monday because I think they prefer not use destructuring in the render. I like it but I think they have this as an standard so maybe next Monday we can check it

Reply:
was discussed we will talk about it in the next technical meeting.